.travis.yml

```diff
@@ -33,8 +34,9 @@ deploy:
   api_key:
     secure: nxaqYrkXLGL3W20/eCnf63DLjMrQAhEuW44jggh1/nI383goa+u6w0bBtWCxRdVzos7t4dpVfS6+kv6oIHacm9zVA+RYrqy5opzCJhq8lmXVVRijbALzUeiFif2HURMaKWj0ynRNVlAyBHzazPTLZVWywifpdSubSkuMWkl20cmuKu/Hg3c1EC9se3OYhhTHx3Hya7xSrctrDEYLsEBAUZzkKfscqRVqwwltS88CgIMtRISDpSBGrtH0t1uAH6NitTSguGgb+QEpqnELcRLymX2G1yzMA4Xr5c/L34MfbBKf8vIuG9t411xYuLoyKoUbroTWxSnPwlSy6PHz+QJ7UCXbDkATOGO3chxlKxglppvI/G3n2YP5Zf2dAaDlHblpvarh55i/4i4sKB2AbvvzkIHrQJwUgmLCbpN8/Vp9GWcGkd6i5U7F8tNInCs6ttX3oGvGOfYEXs02Ctyiea4LAqk4S7GZTuV2QXqxXglL4eRIwZ4UETiwgoAAtHma63Eq7+9t2ykMlk7zAK96FGwJrB97wa08aPuSxL94IYEBmn9Ht/vKXRiNQMvpnfp4rWQtL3cqbVyYAg5EjKb4PsBmnb91+RXtnWFOY1RpZGt8sPXYd+KZYzN1BXTFJEpaLLsIDN6r7nMcAvJDUmucaM+m7giPXz1ZBGAic3UBM1qMCgI=
   file_glob: true
-  file: 
+  file:
     - game-core/build/artifacts/*
+    - game-core/build/libs/triplea-$ENGINE-VERSION-all.zip 
```

Comment:
This file does not appear to exist.

Reply:
The shadow jar builds it.

Reply:
This line was leaked initially, experimentation. I was intending to submit this on a different PR; but in the end I will include this here so we would start publishing the shadow jar file. I think it will be useful for some variations of testing.

Reply:
I ran both the `releaseInstallers` and `releaseAllPlatformsJar` tasks, and all I see is the following:

```
$ ls game-core/build/libs/
triplea-1.9.0.0.dev-all.jar
```

I think you meant to use a `.jar` suffix instead of `.zip`, and the `ENGINE_VERSION` environment variable instead of `ENGINE-VERSION`.

BTW, didn't we [just decide](https://github.com/triplea-game/triplea/issues/2917#issuecomment-362184775) to remove the shadow jar from the release artifacts?  If you want to add it back, I would suggest simply reverting #2946.  Otherwise, you'll need to update _.travis/generate_artifact_checksums_ to account for an artifact outside the _game-core/build/artifacts_ folder.  (Note that the checksums for the lobby server artifact have been broken since #3232.)

Reply:
Let's remove shadow - YAGNI  FTW; one less thing to test. Thanks for staying on point @ssoloff

Reply:
> There may be other consequences to losing the build number from the full version.

I do recall wiring in the game engine to see the full version. There is basically a special method called 'toFullVersion' (or something similar) that prints the full version. Under the hood it's `Version.java` that does everything, it only looks at the first 4 numbers.

We do though *want* to display the full version number, helps with error reporting and knowing which version is which.